Sources/SwifterSwift/UIKit/UIImageExtensions.swift

```diff
@@ -102,11 +102,23 @@ public extension UIImage {
     func scaled(toHeight: CGFloat, opaque: Bool = false) -> UIImage? {
         let scale = toHeight / size.height
         let newWidth = size.width * scale
-        UIGraphicsBeginImageContextWithOptions(CGSize(width: newWidth, height: toHeight), opaque, self.scale)
-        draw(in: CGRect(x: 0, y: 0, width: newWidth, height: toHeight))
-        let newImage = UIGraphicsGetImageFromCurrentImageContext()
-        UIGraphicsEndImageContext()
-        return newImage
+        let size = CGSize(width: newWidth, height: toHeight)
+        let rect = CGRect(origin: .zero, size: size)
+
+        #if !os(watchOS)
+        if #available(tvOS 10.0, *) {
```

Comment:
We have iOS 10 as minimum target deployment for the project, which personally think is too old, but I think we can just drop the old code and remove the availability checks across. Maybe even move tvOS to 10 as well. But that is another discussion